pywbem_mock/_defaultinstanceprovider.py

```diff
@@ -55,7 +65,7 @@
 :meth:`~pywbem_mock.InstanceWriteProvider.CreateInstance` in the superclass to
 complete submission of the ``NewInstance``.
 
-2. Define registraton of the user provider using
+2. Register the user provider using
 :meth:`~pywbem_mock.WBEMConnection.register_provider` to define the namespaces
```

Comment:
Fix the class name in this reference to `FakedWBEMConnection`.

Reply:
DONE